android/src/main/java/com/swmansion/gesturehandler/core/HoverGestureHandler.kt

```diff
@@ -11,6 +11,7 @@ import com.swmansion.gesturehandler.react.RNViewConfigurationHelper
 class HoverGestureHandler : GestureHandler<HoverGestureHandler>() {
   private var handler: Handler? = null
   private var finishRunnable = Runnable { finish() }
+  var stylusData: StylusData = StylusData(0.0, 0.0, 0.0, 0.0, -1.0)
```

Comment:
```suggestion
  var stylusData: StylusData = StylusData(0.0, 0.0, 0.0, 0.0, -1.0)
    private set
```

And please move the magic numbers to be the default values in the constructor.

Reply:
Done in [b8203f2](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/b8203f29171ad5ff49d1ce0b4e1dfa0a6b2f717c) and [4034348](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/40343489095e239b3920e931b22c24c5879115c8)